config/locales/views/assignments/en.yml

```diff
@@ -28,6 +29,22 @@ en:
       description: "Starter code files you upload will be automatically added to every new group's repositories."
       overwrite: "Overwrite group files"
       title: "Starter Code"
+    timed:
+      start_button: "Start"
+      end_time: "End time"
+      start_time_instructions: "You may start any time after: "
+      end_time_instructions: "You must complete this timed assessment before the final due date: "
+      past_due_date: "You have completed this timed assessment."
+      past_end_time: "You cannot start this timed assessment at this time."
+      before_start_instructions: "Once this timed assessment has been started, you will have %{duration_string} to complete it without penalty."
```

Comment:
Note: this message is currently inconsistent with the `duration_string` passed to it, which *includes* penalty periods. Given the other comments I've made, it might be better to rework/remove this string altogether, but I'm drawing your attention to it here in case we want to discuss further.

Reply:
I'm still thinking that the "Once this assessment has been started" section is good information even if we're now repeating the duration numbers themselves later on so I'm inclined to leave this.

I've fixed the value though so that it doesn't include penalty periods. Thanks for catching that.

Reply:
Hmm yes okay, I'm good with keeping this string.